CRITICAL - 2021-03-18 05:39:08 --> preg_match(): Delimiter must not be alphanumeric or backslash
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_match(): D...', '/var/www/html/P...', 10, Array)
#1 /var/www/html/PKL/app/Controllers/Absensi.php(10): preg_match('Linux', 'Mozilla/5.0 (X1...')
#2 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Absensi->index()
#3 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Absensi))
#4 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#6 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#7 {main}
CRITICAL - 2021-03-18 07:23:12 --> Undefined offset: 1
#0 /var/www/html/PKL/app/Controllers/Absensi.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', '/var/www/html/P...', 17, Array)
#1 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Absensi->index()
#2 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Absensi))
#3 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#6 {main}
ERROR - 2021-03-18 13:07:42 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:07:42 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Profile.php(23): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Profile->fetch_data()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profile))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:07:47 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:07:47 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Profile.php(23): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Profile->fetch_data()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Profile))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:25 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:25 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:36 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:36 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:37 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:37 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:37 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:37 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:37 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:37 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:37 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:37 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:37 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:37 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:42 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:42 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:44 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:44 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:44 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:44 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:44 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:44 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:08:50 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:08:50 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:18 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:18 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:24 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:24 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:24 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:24 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:25 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:25 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:25 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:25 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:09:25 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:09:25 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:18 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:18 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:20 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:20 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:20 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:20 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:20 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:20 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:21 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:21 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:10:21 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:10:21 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:02 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:02 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:02 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:02 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:42 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:42 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:43 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:43 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:43 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:43 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:43 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:43 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:56 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:56 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:57 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:57 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:58 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:58 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:58 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:58 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:58 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:58 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:58 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:58 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:58 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:58 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:11:59 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:11:59 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:10 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:10 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:11 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:11 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:11 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:11 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:11 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:11 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:11 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:11 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:11 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:11 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:12 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:12 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:13 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:13 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:14 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:14 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:14 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:14 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:50 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:50 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:51 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:51 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
ERROR - 2021-03-18 13:13:51 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: NO)
CRITICAL - 2021-03-18 13:13:51 --> Unable to connect to the database.
#0 /var/www/html/PKL/system/Database/BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 /var/www/html/PKL/system/Database/BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /var/www/html/PKL/system/Model.php(836): CodeIgniter\Database\BaseBuilder->get()
#3 /var/www/html/PKL/app/Controllers/Auth.php(31): CodeIgniter\Model->__call('get', Array)
#4 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Auth->validasi()
#5 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#9 {main}
CRITICAL - 2021-03-18 13:52:31 --> Undefined variable: j
#0 /var/www/html/PKL/app/Views/user/absensi/laporan/read.php(26): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/var/www/html/P...', 26, Array)
#1 /var/www/html/PKL/system/View/View.php(220): include('/var/www/html/P...')
#2 /var/www/html/PKL/system/View/View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/PKL/system/Common.php(1217): CodeIgniter\View\View->render('user/absensi/la...', Array, true)
#4 /var/www/html/PKL/app/Controllers/Absensi.php(141): view('user/absensi/la...', Array)
#5 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Absensi->fetch_data()
#6 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Absensi))
#7 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#10 {main}
CRITICAL - 2021-03-18 13:52:32 --> Undefined variable: j
#0 /var/www/html/PKL/app/Views/user/absensi/laporan/read.php(26): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/var/www/html/P...', 26, Array)
#1 /var/www/html/PKL/system/View/View.php(220): include('/var/www/html/P...')
#2 /var/www/html/PKL/system/View/View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/PKL/system/Common.php(1217): CodeIgniter\View\View->render('user/absensi/la...', Array, true)
#4 /var/www/html/PKL/app/Controllers/Absensi.php(141): view('user/absensi/la...', Array)
#5 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Absensi->fetch_data()
#6 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Absensi))
#7 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#10 {main}
CRITICAL - 2021-03-18 13:52:33 --> Undefined variable: j
#0 /var/www/html/PKL/app/Views/user/absensi/laporan/read.php(26): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/var/www/html/P...', 26, Array)
#1 /var/www/html/PKL/system/View/View.php(220): include('/var/www/html/P...')
#2 /var/www/html/PKL/system/View/View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/PKL/system/Common.php(1217): CodeIgniter\View\View->render('user/absensi/la...', Array, true)
#4 /var/www/html/PKL/app/Controllers/Absensi.php(141): view('user/absensi/la...', Array)
#5 /var/www/html/PKL/system/CodeIgniter.php(936): App\Controllers\Absensi->fetch_data()
#6 /var/www/html/PKL/system/CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Absensi))
#7 /var/www/html/PKL/system/CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/PKL/public/index.php(44): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/PKL/system/Commands/Server/rewrite.php(45): require_once('/var/www/html/P...')
#10 {main}
